style.css

```diff
@@ -1,564 +1,563 @@
 * {
-    margin: 0;
-    padding: 0;
-    box-sizing: border-box;
-  }
-  
-  html {
-    font-size: 62.5%;
-  }
-  
-  body {
-    font-family: 'Lato', sans-serif;
-    overflow-x: hidden;
-  }
-  
-  :root {
-    --main-color: #ec5242;
-  }
-  
-  /* ***************** */
-  
-  /* **** NAVBAR ***** */
-  
-  /* ***************** */
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+body {
+  font-family: 'Lato', sans-serif;
+  overflow-x: hidden;
+}
+
+:root {
+  --main-color: #ec5242;
+}
+
+/* ***************** */
+
+/* **** NAVBAR ***** */
+
+/* ***************** */
+.header-nav {
+  display: none;
+}
+
+.icon {
+  display: none;
+  height: 6.4rem;
+}
+
+.container {
+  padding: 2.5rem;
+}
+
+.navbar {
+  background-color: #e9ecef;
+}
```

Comment:
Kindly update your navbar to match the behance design better🙏

Your design:
<img width="948" alt="image" src="https://github.com/kkmanuu/Capstone-1/assets/37116322/ac7fe7a9-2cf4-4471-b68b-3252d6338467">

Behance design:
<img width="750" alt="image" src="https://github.com/kkmanuu/Capstone-1/assets/37116322/0aacfeaa-7de7-4dce-b619-3bc0db2f8969">

A few things to note:
- Please increase the spacing between the logo and the nav links
- Please make sure to vertically align the links in the top black bar and the links in the bottom white bar to the right